test/hybrid2d.jl

```diff
@@ -366,3 +366,53 @@ end
     @test norm(exact_hessian .- vec_diff) ≤ 1e-2
 
 end
+
+
+
+@testset "curl" begin
+    hv_center_space, hv_face_space =
+        hvspace_2D(xlim = (-pi, pi), zlim = (-pi, pi))
+
+    ccoords = Fields.coordinate_field(hv_center_space)
+    fcoords = Fields.coordinate_field(hv_face_space)
+
+    u =
+        Geometry.transform.(
+            Ref(Geometry.Covariant1Axis()),
+            Geometry.UVector.(sin.(ccoords.x .+ 2 .* ccoords.z)),
+        )
+    w =
+        Geometry.transform.(
+            Ref(Geometry.Covariant3Axis()),
+            Geometry.WVector.(cos.(3 .* fcoords.x .+ 4 .* fcoords.z)),
+        )
+
+    curl = Operators.Curl()
+    curlC2F = Operators.CurlC2F(
+        bottom = Operators.SetValue(Geometry.Covariant1Vector(0.0)),
+        top = Operators.SetValue(Geometry.Covariant1Vector(0.0)),
+    )
```

Comment:
Are these BCs for the boundary-normal, or boundary-tangent, component of the input vector? (or both?)

Reply:
It's the boundary-tangent component.

Reply:
Or, more to the point, only the boundary-tangent component has any effect.